schema.sql

```diff
@@ -93,4 +93,14 @@ CREATE TABLE visits (
   CONSTRAINT fk_vets
   FOREIGN KEY(vets_id)
   REFERENCES vets(id)
-);
\ No newline at end of file
+);
+
+-- Vet clinic database: database performance audit
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX idx_visits_animal_id ON visits (animal_id);
+CREATE INDEX idx_visits_vets_id ON visits (vets_id);
+
+CREATE INDEX idx_owners_email ON owners (email);
```

Comment:
## Optional

- [ ] I suggest making the index on the case insensitive version of the email, 
As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:
`CREATE INDEX ON owners ((lower(email)));`

Reply:
Hi @DeliceLydia,
Wow!!!!. These will be very helpful.

Reply:
done